i2c/oled_i2c/README.adoc

```diff
@@ -0,0 +1,68 @@
+= Attaching an OLED display via I2C
+
+This example code shows how to interface the Raspberry Pi Pico with an 128x32 OLED display board based on the SSD1306 display driver.
+
+The code displays a series of tiny raspberries that scroll horizontally, in the process showing you how to initialize the display, write to the entire display, write to only a portion of the display, and configure scrolling.
+
+In contrast to the sensors in the examples library, the SSD1306 is not operated via direct reads/writes to registers. Instead, a list of versatile commands is defined (see datasheet) that allows the user to access all the capabilities of the driver. Ultimately, these commands end up writing to internal registers but the key point is that the I2C write process is now different. After sending a slave address, the data that follows can be either a command, flags to follow up a command or data to be written directly into the display's RAM. A control byte is required for each write after the slave address so that the driver knows what type of data is being sent.
+
+This display is 32 pixels high by 128 pixels wide. These 32 pixels are partitioned into 4 pages, each 8 bits in height. In RAM, This looks roughly like:
+
+----
+           | COL0 | COL1 | COL2 | COL3 | ...
```

Comment:
Does this actually extend out to 128 columns? If so, that should perhaps be clarified like:
```
           | COL0 | COL1 | COL2 | COL3 | ... | COL126 | COL127 |
```
?
And perhaps "These 32 pixels" should be "These 32 vertical pixels" ? And "8 bits in height" -> "8 pixels in height" ? :shrug:

Reply:
Much clearer!